README.md

```diff
@@ -58,17 +58,21 @@ load_regs/fec_reg_parts.txt.
 If you are loading regs for a new year, you will need to reset the database. To do that, run:
 ```bash
 $ cf unbind-service eregs fec-eregs-db
+$ cf service-keys fec-eregs-db
+$ cf delete-service-key fec-eregs-db [name of service key from previous]
 $ cf delete-service fec-eregs-db
 $ cf create-service aws-rds shared-psql fec-eregs-db
+$ cf bind-service eregs fec-eregs-db
 $ cf restage eregs
 ```
 
 Now you can load the regs with:
 
 ```bash
-$ python load_regs/load_fec_regs.py dev $HTTP_AUTH_USER $HTTP_AUTH_PASSWORD
+$ python load_regs/load_fec_regs.py [env] $HTTP_AUTH_USER $HTTP_AUTH_PASSWORD
```

Comment:
Minor suggestion - call the placeholder `space` as that's what they are called in Cloud Foundry.